plugins/modules/foreman_host.py

```diff
@@ -150,6 +171,8 @@ def main():
         if 'hostgroup' not in entity_dict and entity_dict.get('managed', True):
             module.fail_json(msg='Hostgroup can be omitted only with managed=False')
 
+        entity_spec = module.handle_common_host_spec(entity_dict)
```

Comment:
```suggestion
        entity_dict = module.handle_common_host_spec(entity_dict)
```

Reply:
(will also fix the linter error ;))
```
plugins/modules/foreman_host.py:174:9: F841 local variable 'entity_spec' is assigned to but never used
```

Reply:
(and that's why I wanted tests ;P)

Reply:
(Yes, but the linter saved us. And it's tagged WIP)